### PR TITLE
added Tilemap, house and fox

### PR DESCRIPTION
**Pull request by dean-rakic**
- implemented tilemap
- added house scene with dissapearing front
- added fox item

### CODE REVIEW

File: inventory/inv_scripts/inventory_slot.gd [hunk 1]
```diff
@@ -13,3 +13,5 @@ func update(slot: InventorySlot):
 		if slot.amount > 1:
 			amount_text.visible = true
 		amount_text.text = str(slot.amount)
+		if slot.item.name == "fox":
+				item_visual.scale = Vector2(0.075, 0.075)
```
Comment (marinprusac) [inline inventory/inv_scripts/inventory_slot.gd]: Number of indentations?

File: scenes/area_2d.gd
Comment (marinprusac) [inline scenes/area_2d.gd:1]: Surely there's a better way to name this script?

File: sprites/Axe.png
Comment (marinprusac) [inline sprites/Axe.png:1]: We already have an axe.png in sprites/misc.

File: sprites/Debries.png.import
Comment (marinprusac) [inline sprites/Debries.png.import:1]: We already have a debris.png in sprites/misc.

File: sprites/Key.png
Comment (marinprusac) [inline sprites/Key.png:1]: We already have a key.png in sprites/misc.

File: sprites/Plank.png
Comment (marinprusac) [inline sprites/Plank.png:1]: We already have a plank.png in sprites/misc.

File: sprites/TileSet.png
Comment (marinprusac) [inline sprites/TileSet.png:1]: Already have this in sprites/tiles. No?

**Review by marinprusac — CHANGES_REQUESTED** [review]
There's some redundant asset importing among other things.
We should organize our artwork a little better so it doesn't get cluttered.

File: scenes/area_2d.gd
Comment (savipauk) [reply scenes/area_2d.gd:1]: +1 please rename

**Review by marinprusac — APPROVED** [review]

**Pull request merged**